examples/README.md

```diff
@@ -0,0 +1,60 @@
+# examples
+
+This directory contains example code for both installing Contour and its associated Envoy Pods under various conditions, and also for setting up various extensions and other pieces of software that may help with running Contour.
```

Comment:
Split the sentence after  under various conditions,

s/  under various conditions// jsut say "installing Contour and Envoy"